mgmtsystem_probability/model/mgmtsystem_probability.py

```diff
@@ -0,0 +1,47 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 - Present
+#    Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.tools.translate import _
+from osv import fields, orm
+
+_CATEGORIES = [
+    ("hazard", _("hazard")),
+    ("security", _("security")),
+]
```

Comment:
I'm uneasy with this being repeated in the modules.
It it's to be "used across different modules", maybe it could make sense to have a central table where the area specific modules would add the needed records.
Just an idea though... could be something to improved later.